### ISSUE TITLE
Recognize nothogenus ×Cyclobotrya in the Dryopteridaceae, and to keep Cyclosorus and Polybotrya as distinct

### ISSUE THREAD

**Issue by schwartsburd** [pteridogroup/ppg#21]
### Name of taxon

×Cyclobotrya

### Rank of taxon

Nothogenus

### Approximate number of species affected

2

### Description of change

Cyclosorus and Polybotrya are two Neotropical genera with stable circumscriptions since the 1980s, after their respective taxonomic revisions published by Smith (Amer. Fern J. 76: 56–98. 1986) and Moran (Illinois Nat. Hist. Survey Bull. 34: 1–138. 1987). They show very distinct morphology: Cyclosorus has meristeles without schlerenchyma sheaths, the leaves are sub-dimorphic, with the fertile laminae slightly more contracted, and the discrete sori are indusiate (peltate or round-reniform); whereas Polybotrya has meristeles surrounded by schlerenchyma sheaths, the leaves are strongly dimorphic, with the fertile laminae skeletonized, the sori are somehow primarily discrete, but due to the skeletonized laminae, they can get an acrostichoid appearance (sometimes forming coenosori, and sometimes getting to be amfiacrostichoid), and they are never indusiate.
	A comprehensive phylogeny performed on them (Moran & Labiak, Int. J. Plant Sci. 176: 880–891. 2015) showed them to be sister genera and also showed many apomorphies for each one, such as habit (terrestrial vs. terrestrial root climber), the schlerenchyma sheaths around the meristeles in Polybotrya, indusia (present vs. absent), the peltate indusium attachment in Cyclosorus, and the strong leaf dimorphism in Polybotrya. The two genera are also equally sister to Maxonia.
	In 2016, Engels & Canestraro collected a specimen with undoubtfully intermediate morphology between those two genera and co-occurring with both Cyclodium meniscioides and Polyborya goyazensis. The specimen had also malformed sporangia and spores. Thus, Engels & Canestraro (Brittonia 69: 307–312. 2017) published the new nothogenus ×Cyclobotrya, along with the inter-generic hybrid ×Cyclobotrya telespirensis.
	Right after Engels & Canestraro (2017) paper, Schwartsburd and Prado et al. re-examined some specimens they had collected in 2011 and realized they also had collected a strange specimen with undoubtfully intermediate morphology between those two genera. The specimen had even an acrostichoid sori with indusia! and also malformed sporangia and spores. The specimen grew along populations of Cyclosorus meniscioides and Polybotrya osmundacea. Thus, Schwartsburd and Prado et al. (Brittonia 70: 25–30. 2018) published a second hybrid species in the nothogenus: ×Cyclobotrya amalgamata.

### Reason for change

Although there has not been any attempt to genetically test these hybrids, their intermediate morphology leaves no doubt about their identity. The thing is: Cyclosorus and Polybotrya are sister genera. Therefore, the lumping of these two genera into Polybotrya s.l. is an option, or we keep the two genera apart (Cyclosorus and Polybotrya) and also consider valid the nothogenus ×Cyclobotrya.
We here propose the adoption of the last solution.

### Reference(s) for publication of the name

Engels, M.E. & Canestraro, B.K. 2017. ×Cyclobotrya: a new hybrid genus between Cyclodium and Polybotrya (Dryopteridaceae) from the Brazilian Amazon. Brittonia 69: 307–312. 2017. http://dx.doi.org/10.1007/s12228-017-9468-2 

Moran, R.C. 1987. Monograph of the Neotropical fern genus Polybotrya (Dryopteridaceae). Illinois Nat. Hist. Survey Bull. 34: 1–138.

Moran, R.C. & Labiak, P.H. 2018. Phylogeny of the polybotryoid fern clade (Dryopteridaceae). Int. J. Plant Sci. 176: 880–891. http://dx.doi.org/10.1086/683393 

Schwartsburd, P.B., Canestraro, B.K., Moran, R.C., Prado, J. & Smith, A.R. 2018. A second ×Cyclobotrya (Dryopteridaceae) from Brazil. Brittonia 70: 25–30. http://dx.doi.org/10.1007/s12228-017-9501-5

Smith, A.R. 1986. Amer. Fern J. 76: 56–98.

### Code of Conduct

- [X] I agree to follow the PPG Code of Conduct

**Comment by schwartsburd** [comment]
By the way, this proposal was written by me and Jefferson Prado. I thought there would be a place to include the authors...

**Comment by joelnitta** [comment]
@schwartsburd Good point - sorry that wasn't included in the template! It has now been added, and I edited your proposal to include your names. Is that better?

**Comment by susanfawcett** [comment]
I think this proposal needs to be rewritten as "Recognize nothogenus ×Cyclobotrya in the Dryopteridaceae, and to keep Cyclodium and Polybotrya as distinct"

Cyclosorus (Thelypteridaceae) is not closely related to Polybotrya and does not hybridize with it.

**Comment by schwartsburd** [comment]
Dear Susan: you are right! Dear all: I am terribly sorry. I could edit the title, but not the text (I don't know how to do it). The erratum is the following: wherever is written "Cyclosorus", please read "Cyclodium".

Am I too busy? Am I getting old? Insane? Sorry for that, people. My bad,

Pedro

**Comment by schwartsburd** [comment]
@joelnitta could you please fix it in template? Thanks

**Comment by susanfawcett** [comment]
@schwartsburd no worries! Everyone does something like this now and again!

**Comment by joelnitta** [comment]
Fixed :)

**Comment by joelnitta** [comment]
This proposal was voted on during PPG Ballot 3 (voting period September 2023). A total of 63 votes were cast. There were 56 'Yes' votes (88.9%) and 7 'No' votes (11.1%). The proposal passes.

**Comment by joelnitta** [comment]
Implemented in Rhakhis: https://list.worldfloraonline.org/rhakhis/ui/index.html#wfo-4100004291
